### DISCUSSION TITLE
Can we get an example where functions and types are shared between front-end code and edge functions?

### DISCUSSION THREAD

**Discussion by carcinocron** [supabase/supabase#26838] [Feature Requests]
I have a pnpm/vite based front-end and am attempting to use supabase functions. I've tried various things like this import_map.json file:

```json
{
  "imports" : {
    "@my-org/shared/": "npm:../shared",
    "@my-org/my-solid-js-app/": "npm:./../my-solid-js-app"
  }
}
```

but it seems like everything you can try fails to `deno lint` correctly but more importantly also fails to deploy correctly. The only thing I've gotten to work is copying and pasting types/functions into the deno part of the repo. The rest of my repo is a standard pnpm workspaces setup, which the non-deno parts are working/integrating smoothly.

**Comment by apostolos-geyer** [comment]
+1
